website/addons/dryad/routes.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+from framework.routing import Rule, json_renderer
+from website.addons.dryad import views
+from website.routes import OsfWebRenderer
+
+TEMPLATE_DIR = '../addons/dryad/templates/'
+
+
+page_routes = {'rules':
```

Comment:
I'm seeing now that you do have code for a Dryad specific pages. At this I don't have a very clear idea of the expectations or featureset of this PR. Would you be willing to update the description with a more detailed description and screenshots of all UI changes?